specification/agfood/data-plane/Microsoft.AgFoodPlatform/preview/2021-03-31-preview/examples/Boundaries_CreateOrUpdate.json

```diff
@@ -0,0 +1,95 @@
+{
+  "parameters": {
+    "farmerId": "FARMER123",
+    "boundaryId": "BOUNDARY123",
+    "api-version": "2021-03-31-preview"
```

Comment:
Probably the issue is that we don't have a body here.

Reply:
Yes, added body and it worked.